### DISCUSSION TITLE
Plugins on run

### DISCUSSION THREAD

**Discussion by kaihendry** [simonw/datasette#2088] [Q&A]
Hi, I want plugins like:
1. datasette-json-html
2. datasette-block-robots

Via https://github.com/kaihendry/aws-partners-datasette/blob/main/requirements.txt but they are not shown in my `make run` http://127.0.0.1:8001/-/plugins ... I've only managed to get the plugins loading via a publish target in my https://github.com/kaihendry/aws-partners-datasette/blob/main/Makefile 

The problem is that I want to see the plugins in action locally in my run target, not on the remote Cloud service.

# I've tried

Adding plugins to https://github.com/kaihendry/aws-partners-datasette/blob/main/metadata.yaml but appears to have no affect?

**Comment by simonw** [comment]
It looks like they're working on your deployed instance: https://aws-partners.fly.dev/-/plugins

My best guess is that they aren't installed on your laptop in the same virtual environment as Datasette itself.

I suggest trying this:
```bash
datasette install datasette-block-robots datasette-json-html
```
Then run this to confirm they were installed correctly:
```bash
datasette plugins
```

**Comment by kaihendry** [comment]
Sorry, I'm super unfamiliar with Python. I got it working on my nix system like so:

```
python -m venv .venv
source .venv/bin/activate
pip install -r requirements.txt
```

Thank you!
